cmd/network/verification.go

```diff
@@ -553,20 +553,12 @@ func (e *EgressVerification) getSecurityGroupId(ctx context.Context) (string, er
 				Name:   aws.String("tag:Name"),
 				Values: []string{fmt.Sprintf("%s-default-sg", e.cluster.ID())},
 			},
-			{
-				Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.ID())),
-				Values: []string{"owned"},
-			},
 		}
 	default:
 		filters = []types.Filter{
 			{
 				Name:   aws.String("tag:Name"),
-				Values: []string{fmt.Sprintf("%s-master-sg", e.cluster.InfraID())},
-			},
-			{
-				Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.InfraID())),
-				Values: []string{"owned"},
+				Values: []string{fmt.Sprintf("%s-master-sg", e.cluster.InfraID()), fmt.Sprintf("%s-controlplane", e.cluster.InfraID())},
```

Comment:
Add a comment so we can deprecate code later as clusters age up?  We won't have the mental context for this in years++, and will end up carrying this forward without knowing why.

```suggestion
				// Prior to 4.16: <infra_id>-master-sg
				// 4.16+: <infra_id>-controlplane
				Values: []string{fmt.Sprintf("%s-master-sg", e.cluster.InfraID()), fmt.Sprintf("%s-controlplane", e.cluster.InfraID())},
```

Reply:
+1 to this, great suggestion

Reply:
It may be hard to ever fully deprecate this, as clusters could technically be installed on a version way before 4.16 and then update beyond. In that case, the security group tags would still stick with their installation values.

_That said_, I still like this comment anyway 🙂